frontend/src/plugins/api.js

```diff
@@ -24,6 +66,11 @@ class API {
             return null;
         }
 
+        if (response.status === 401 && this.token) {
+            // Token has expired. Automatic log out.
+            await this.logout();
+        }
```

Comment:
In the future we might want to automatically renew the token :thinking: 

-> Create a separate issue.

Reply:
#38